### ISSUE TITLE
Can't apply integer value to enum attribute with theme

### ISSUE THREAD

**Issue by wallacen60** [robolectric/robolectric#953]
Robolectric does not handle applying an integer value to an enum attribute using a theme. However, Android does support this. Checkout this example project:

https://github.com/wallacen60/CrashOnEnum

Verified w/ robolectric-2.3-20140207.154407-115.jar

Causes this exception chain: 
```
android.view.InflateException: XML file ...app/build/res/all/debug/layout/main.xml line #-1 (sorry, not yet implemented): Error inflating class com.example.view.MyTextView
	at android.view.LayoutInflater.createView(LayoutInflater.java:613)
...
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
...
Caused by: java.lang.RuntimeException: error converting 2 using EnumConverter
	at org.robolectric.shadows.Converter.convertAndFill(Converter.java:150)
...
Caused by: java.lang.RuntimeException: no value found for 2
	at org.robolectric.shadows.Converter$EnumOrFlagConverter.findValueFor(Converter.java:375)
```

The workaround is to apply the value through another style, like how TextView/textAppearance is done in the framework.

**Comment by jaredsburrows** [comment]
Without modifying your code, I am seeing this:

com.example.app.test.MyTextViewTest > testOne FAILED
    org.junit.ComparisonFailure: expected:<[Hello world]> but was:<[2]>
        at org.junit.Assert.assertEquals(Assert.java:115)
        at org.junit.Assert.assertEquals(Assert.java:144)
        at com.example.app.test.MyTextViewTest.testOne(MyTextViewTest.java:26)

**Comment by dlubarov** [comment]
I saw the same issue using 3.0. It's intermittent though. So far I've only seen it on a couple CI machines, so I haven't had much luck investigating.

    Caused by: java.lang.RuntimeException: no value found for WARNING at org.robolectric.shadows.Converter$EnumOrFlagConverter.findValueFor(Converter.java:431)
       ...

**Comment by jaredsburrows** [comment]
@erd @xian @jongerrish

**Comment by xian** [comment]
Fixed with 3.0(ish).
